openmdao/docs/features/building_blocks/components/metamodelunstructured_comp.rst

```diff
@@ -86,4 +86,24 @@ to be the same size as specified in the ``vec_size`` argument.
     openmdao.components.tests.test_meta_model_unstructured_comp.MetaModelTestCase.test_metamodel_feature_vector2d
     :layout: code, output
 
+
+Using Surrogates That Do Not Define Linearize Method
+----------------------------------------------------
+
+In some cases, users might define surrogates but not define a `linearize` method. In this case, the
```

Comment:
In general, I think that the docs address the user (the reader) rather than referring to them in the third person.  (e.g. the intro to this section: "MetaModelUnStructuredComp lets you quickly create a component").  Suggest replacing references to "users" here and "the user" below with the pronoun "you" for consistency. (also removing the "user" reference in the embedded test)

Reply:
All comments by swryan addressed